Main/Source/miscitem.cpp

```diff
@@ -3656,3 +3748,209 @@ alpha ullrbone::GetOutlineAlpha(int Frame) const
   Frame &= 31;
   return 50 + (Frame * (31 - Frame) >> 1);
 }
+
+material* trinket::RemoveMaterial(material* Material)
+{
+  if(GetConfig() == DEAD_FISH)
+  {
+    item* Bones = trinket::Spawn(BONE_FISH);
+    DonateSlotTo(Bones);
+    DonateIDTo(Bones);
+    SendToHell();
+    return 0;
+  }
+  else
+    return item::RemoveMaterial(Material);
+}
+
+truth trinket::Necromancy(character*)
+{
+  if(GetConfig() == BONE_FISH)
+  {
+    GetSlot()->AddFriendItem(trinket::Spawn(DEAD_FISH));
+    RemoveFromSlot();
+    SendToHell();
+    return true;
+  }
+  else
+    return false;
+}
+
+truth trinket::RaiseTheDead(character*)
+{
+  if(GetConfig() == BONE_FISH)
+  {
+    ADD_MESSAGE("%s suddenly comes back to life, but quickly air-drowns again.", CHAR_NAME(DEFINITE));
```

Comment:
I think "suffocates" would be better

Reply:
Why not. :) "Air-drowns" sounds cool to me, but "suffocates" is probably more normal.